sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreProvider.scala

```diff
@@ -365,6 +435,10 @@ private[sql] class RocksDBStateStoreProvider
   private val keyValueEncoderMap = new java.util.concurrent.ConcurrentHashMap[String,
     (RocksDBKeyStateEncoder, RocksDBValueStateEncoder)]
 
+  private val colFamilyNameToIdMap = new java.util.concurrent.ConcurrentHashMap[String, Short]
+  // TODO SPARK-48796 load column family id from state schema when restarting
```

Comment:
This sounds like we can't release before addressing SPARK-48796, do I understand correctly? If then I'd need to mark the ticket as blocker for Spark 4.0.0.

Reply:
Also do we have a path forward on storing column family id? Will SPARK-48796 address this altogether?

Reply:
Yes correct - this will be handled in that ticket where we store the vcf id within the new state schema format